src/phoenix/session/session.py

```diff
@@ -327,6 +333,10 @@ def launch_app(
     default_umap_parameters: Dict[str, Union[int, float]], optional, default=None
         User specified default UMAP parameters
         eg: {"n_neighbors": 10, "n_samples": 5, "min_dist": 0.5}
+    notebook_environment: NotebookEnvironment, optional, default=None
+        The environment the notebook is running in. This is either 'local', 'colab', or 'sagemaker'.
```

Comment:
nit. At first glance this looks like it's taking a `str` value. Given the dynamic nature of python, it wouldn't be too bad we support that also (e.g. pandas takes str arguments all the time, and the usability of `str` is quite nice). 
```python
if isinstance(notebook_environment, str):
    nb_env: NotebookEnvironment = NotebookEnvironment(notebook_environment)
else:
    nb_env = notebook_environment
```

Reply:
maybe we should add this as one of our environment variables too (which would be `str`)

Reply:
Good call